content/_includes/page/recents.njk

```diff
@@ -3,14 +3,16 @@
   'Podcast',
   'Video',
   'Link',
-  'Case Study'
+  'Case Study',
+  'Winging It'
 ] -%}
 {%- set tag_ribbon_mapping = {
   'Article': 'Articles',
   'Podcast': 'Podcasts',
   'Video': 'Videos',
   'Link': 'Links',
-  'Case Study': 'Case Studies'
+  'Case Study': 'Case Studies',
+  'Winging It': 'Episodes'
```

Comment:
@stacyk I fixed this in https://github.com/oddbird/oddleventy/commit/6fb5bd9b3b32ed4eb329b031d189ba9eeb2ccd0c, simplified the logic a bit and made a similar change for the other types of posts (so that e.g. on `/tags/podcast/` it says "Podcasts" instead of "Posts").